scaffold/convert/header.template.js

```diff
@@ -1,4 +1,7 @@
-<% if (before && !session && !oauth) { %>const maybeIncludeAuth = (request, z, bundle) => {
+<% if (customBasic) { %>
```

Comment:
I use the term "custom basic" to indicate a basic authentication that does not use username and password as user-facing authentication fields.